docs/containers/bridge-to-kubernetes.md

```diff
@@ -163,6 +163,8 @@ For Node.js services, you can use code similar to the following, taken from the
 
  First, retry the activation using the button. If it repeatedly does not succeed, see [https://github.com/microsoft/mindaro/issues/32](https://github.com/microsoft/mindaro/issues/32).
 
+If EndpointManager fails when you are running as a normal user, or when you are using Bridge to Kubernetes in a remote SSH session, the issue might be that Bridge to Kubernetes cannot modify the hosts file due to a permissions issue. To enable remote SSH or running as normal user, you should update your code to use Kubernetes service environment variables, and configure VS Code to use them, as described [here](kubernetes-env-vars.md).
```

Comment:
@rakeshvanga Aren't we blocking the scenarios so that users cannot fall into them if they didn't set the env variable? I don't want the users to assume all EndpointManager issues are due to env vars, and spend time to fix it if it's actually something else.

Reply:
No, we are not blocking the endpoint manager errors that are related to elevated permission issues at the moment. It will be categorized in the next release. 
@ghogen Can we update this text to start with `When you are using Bridge to Kubernetes in a remote SSH session,...` ?
Once I complete implementing the logic to categorize EndpointManager  errors, we can update the doc to reflect with exact warning message for EndpointManager  errors.